src/components/AddBook.js

```diff
@@ -18,7 +18,9 @@ const AddBook = () => {
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    dispatch(addBook({ item_id: uuidv4(), title: book.title, author: book.author }));
+    dispatch(postBook({
+      item_id: uuidv4(), title: book.title, author: book.author, category: 'Action',
+    }));
```

Comment:
- [ ] I noticed that when I add a book, the screen doesn't update the new book entry unless I reload the page. Please work on this as it affects the user experience of the app.
- [ ] This issue also occurs when I remove a book. Kindly make it work 👍